suspect/processing/channel_combination.py

```diff
@@ -3,14 +3,19 @@
 
 def svd_weighting(data):
     p, _, v = numpy.linalg.svd(data, full_matrices=False)
+    
     channel_weights = p[:, 0].conjugate()
+    # normalize so they sum to 1
+    channel_weights = channel_weights/numpy.sum(numpy.abs(channel_weights))
 
     # try some basic phase correction
     # in our truncation of the SVD to rank 1, we know that v[0] is our FID
     # use the first point of it to phase the signal
     phase_shift = numpy.angle(v[0, 0])
+    
+    channel_weights_phase_corrected = channel_weights * numpy.exp(-1j * phase_shift)        
 
-    return channel_weights * numpy.exp(-1j * phase_shift) / numpy.sum(numpy.abs(channel_weights))
+    return channel_weights_phase_corrected, channel_weights, phase_shift
```

Comment:
I am surprised that the unsuppressed water phase is not correct for the suppressed signal, they should have an exactly matched phase, except for the suppressed water peak, and who cares about that anyway. It should also be extremely simple to modify the phase of the suppressed signal after using the wrongly phased weights, simply by making the first point of the FID real, which is all this does anyway. I would really prefer to keep this as simple as possible as a function, which means returning just one thing.